array-equals.rb

```diff
@@ -0,0 +1,32 @@
+# CS Fun
+# Array Equals
+
+def array_equals(int_array1, int_array2)
```

Comment:
Nice choice in naming them `int_array`. This clearly expresses what is being tested.